Fix "documemnation" typo

OrionOth opened the pull request:
This adds some GitHub Issue templates we can use to format bugs and feature info better.

steph-lheureux committed:
Update issue templates

liamgilligan committed:
Recreate functional-bug.md

Re-adding bug report template

OrionOth committed:
doc: Add [FCODE] to feature.md Feature Template

steph-lheureux committed:
Update issue templates

Added task issue template

OrionOth committed:
Merge pull request #6 from CabrilloRoboticsClub/issue-template

Update task template

steph-lheureux committed:
Update and rename functional-bug.md to bug.md

steph-lheureux committed:
Add documentation issue

OrionOth committed:
Update tasks.md

making it better

OrionOth committed:
chore: fix spelling in doc template

tinymassi commented on .github/ISSUE_TEMPLATE/documentation.md:
```diff
@@ -0,0 +1,26 @@
+---
+name: Documemnation
```
Change "Documemnation" to "Documentation"

OrionOth committed:
doc: Fix frontmatter value typo

There was a typo in the YAML frontmatter of the documentation request issue template.

OrionOth committed:
Merge branch 'main' into issue-templates

theoparis reviewed (approved):
LGTM.
